src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -1412,15 +1414,156 @@ bool ModControllableAudio::offerReceivedCCToLearnedParams(MIDIDevice* fromDevice
 					}
 				}
 				else {
-					newKnobPos = 64;
-					if (value < 127) {
-						newKnobPos = (int)value - 64;
+					if (midiEngine.midiTakeover == MIDI_TAKEOVER_JUMP) {//Midi Takeover Mode = Jump
+						newKnobPos = 64;
+						if (value < 127) {
+							newKnobPos = (int)value - 64;
+						}
+						knob->previousPositionSaved = false;
+					}
+					else { //Midi Takeover Mode = Pickup or Value Scaling
+						/*
+						MIDI TAKEOVER MODES:
```

Comment:
This high level description probably belongs in community features.md rather than the code

Reply:
I have made the recommended change and moved the Takeover Mode description to community features.md. See most recent commit.